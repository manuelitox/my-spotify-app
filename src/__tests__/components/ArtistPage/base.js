import React from 'react'
import { shallow } from 'enzyme'

import StatelessArtistPage from 'components/ArtistPage/base'

describe('Component | StatelessArtistPage', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StatelessArtistPage />
    )
  })
  it('should render a component', () => {
    expect(component).toMatchSnapshot()
  })
})