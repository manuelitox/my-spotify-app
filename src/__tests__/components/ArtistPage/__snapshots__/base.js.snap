// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StatelessArtistPage should render a component 1`] = `
<styled.div>
  <Header />
  <Banner />
  <Component>
    <Connect(Albums) />
    <Connect(Tracks) />
  </Component>
  <Connect(RelatedArtists) />
</styled.div>
`;
